src/databases.hpp

```diff
@@ -114,7 +114,7 @@ class MultiDatabaseEntry {
     std::vector<unsigned char> bytes;
 };
 
-typedef std::vector<MultiDatabaseEntry> MultiDatabaseDescriptor;
+using MultiDatabaseDescriptor = std::vector<MultiDatabaseEntry>;
```

Comment:
why did you choose to change this one typedef among all the typedefs. There's probably about a dozen other typedefs